### ISSUE TITLE
npm install sha3 is no such file or directory

### ISSUE THREAD

**Issue by jianxuanbing** [phusion/node-sha3#45]
```
I:\YL\Git\bing-nest-seed\node_modules\sha3>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
在此解决方案中一次生成一个项目。若要启用并行生成，请添加“/m”开关。
  addon.cpp
  displayIntermediateValues.cpp
  KeccakF-1600-reference.cpp
  KeccakNISTInterface.cpp
  KeccakSponge.cpp
  win_delay_load_hook.cc
..\src\KeccakNISTInterface.cpp(14): fatal error C1083: 无法打开包括文件: “string.h”: No such file or directory [I:\YL\Git\bing-nest-seed\node_modules\sha3\build\sha3.vcxproj]
..\src\displayIntermediateValues.cpp(14): fatal error C1083: 无法打开包括文件: “stdio.h”: No such file or directory [I:\YL\Git\bing-nest-seed\node_modules\sha3\build\sha3.vcxproj]
c:\users\jianx\.node-gyp\9.3.0\include\node\v8.h(18): fatal error C1083: 无法打开包括文件: “stddef.h”: No such file or directory (编译源文件 ..\src\addon.cpp) [I:\YL\Git\bing-nest-seed\node_modules\sha3\build\sha3.vcxproj]
C:\Program Files (x86)\Windows Kits\8.1\Include\um\windows.h(160): fatal error C1083: 无法打开包括文件: “excpt.h”: No such file or directory (编译源文件 C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\src\win_delay_load_hook.cc) [I:\YL\Git\bing-nest-seed\node_modules\sha
3\build\sha3.v
cxproj]
..\src\KeccakSponge.cpp(14): fatal error C1083: 无法打开包括文件: “string.h”: No such file or directory [I:\YL\Git\bing-nest-seed\node_modules\sha3\build\sha3.vcxproj]
..\src\KeccakF-1600-reference.cpp(14): fatal error C1083: 无法打开包括文件: “stdio.h”: No such file or directory [I:\YL\Git\bing-nest-seed\node_modules\sha3\build\sha3.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` failed with exit code: 1ld.exe` failed with exit code: 1                                    ode_modules\npm\node_modules\node-gyp\lib\build.js:258:23)
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:258:23)          child_process.js:209:12)
gyp ERR! stack     at ChildProcess.emit (events.js:159:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"child_process.js:209:12)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program
Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd I:\YL\Git\bing-nest-seed\node_modules\sha3
gyp ERR! node -v v9.3.0
gyp ERR! node-gyp -v v3.6.2
gyp ERR! not ok
```

**Comment by canterberry** [comment]
Fixed in #44, which removes compilation steps from the build.

**Comment by canterberry** [comment]
Fixed in v2.0.0.
